plugins/org.eclipse.fordiac.ide.model/model/lib.ecore

```diff
@@ -1974,7 +1974,8 @@
   </eClassifiers>
   <eClassifiers xsi:type="ecore:EClass" name="ServiceInterfaceFBType" eSuperTypes="#//FBType"/>
   <eClassifiers xsi:type="ecore:EClass" name="SimpleECAction">
-    <eStructuralFeatures xsi:type="ecore:EReference" name="algorithm" eType="#//Algorithm"/>
+    <eStructuralFeatures xsi:type="ecore:EAttribute" name="algorithm" lowerBound="1"
```

Comment:
Please don't set a `lowerBound` in case there is no algorithm for the action.

Reply:
changed